flow/skia_gpu_object_unittests.cc

```diff
@@ -31,19 +32,133 @@ class TestSkObject : public SkRefCnt {
   fml::TaskQueueId* dtor_task_queue_id_;
 };
 
-TEST_F(SkiaGpuObjectTest, UnrefQueue) {
-  fml::RefPtr<fml::TaskRunner> task_runner = CreateNewThread();
-  fml::RefPtr<SkiaUnrefQueue> queue = fml::MakeRefCounted<SkiaUnrefQueue>(
-      task_runner, fml::TimeDelta::FromSeconds(0));
+// Death tests and threads do not get along.  This fixture gives access to the
+// current thread context but doesn't start any other threads.
+using SkiaGpuObjectDeathTest = ThreadTest;
```

Comment:
I believe you can use `::testing::FLAGS_gtest_death_test_style = "threadsafe";` to get somewhat reasonable behavior. You can add that to a fixture.